test/cookie.test.js

```diff
@@ -2,668 +2,204 @@
 
 const tap = require('tap')
 const test = tap.test
-const Fastify = require('fastify')
-const sinon = require('sinon')
-const cookieSignature = require('cookie-signature')
-const plugin = require('../')
 
-test('cookies get set correctly', (t) => {
-  t.plan(7)
-  const fastify = Fastify()
-  fastify.register(plugin)
+const cookie = require('../cookie')
```

Comment:
The test changes are HUGE

Would not be possible to restore the previous format?
I would expect no changes on tests, but only additions. Am I wrong?

Reply:
Dont worry. I renamed the original cookie.test.js to plugin.test.js. See commit comment 
https://github.com/fastify/fastify-cookie/pull/152/commits/a03c3969c476f74ec393fd1b6c298c6392be2646

The old "cookie.test.js" is now named "plugin.test.js" and matching to the "plugin.js" file. The integrated cookie.js has now the test cases from the old cookie package (adapted to the format of this package), named "cookie.test.js". I did not change a line in the old cookie.test.js/ new plugin.test.js.

Reply:
I don't doubt it, but the `git diff` is too much to afford.

I think the issue is on the `cookie.test.js` file name.
Could you rename it `cookie-module.test.js`?

Then git should understand the file renaming and it will not try to compare the `plugin.test.js` against the new test from the cookie module

Reply:
done